### PR TITLE
feat: update to the new backup method

### PR DESCRIPTION
**Pull request by linuxbandit**
depends on other PR

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -99,17 +99,20 @@ remove-agents:
...
-	./helper.sh --execute postgres-events -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/events' --inserts > events.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	/opt/bu-settings/dump.sh postgres-events
 
 backup_statutory:
-	./helper.sh --execute postgres-statutory -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/statutory' --inserts > statutory.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	/opt/bu-settings/dump.sh postgres-statutory
 
 backup_discounts:
-	./helper.sh --execute postgres-discounts -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/discounts' --inserts > discounts.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	/opt/bu-settings/dump.sh postgres-discounts
```
Comment (serge1peshcoff) [inline Makefile:115]: why the absolute path and not relative?

Comment (linuxbandit) [reply Makefile:115]: explicit is better than implicit

Comment (serge1peshcoff) [reply Makefile:115]: but it won't work if we'll have different folder structure, it's not standardized yet. and it won't work locally (for example I have the docker setup at `/Users/sergeypeshkov/oms-docker`)

Comment (linuxbandit) [reply Makefile:115]: the backup scripts is not depending on the location of myaegee

Comment (serge1peshcoff) [reply Makefile:115]: what do you think of moving it under scripts/ directory, then invoking it as scripts/dump.sh with relative name?

Comment (linuxbandit) [reply Makefile:115]: as i mentioned in the other comment, probably ok, I'll give a thought (have to check the ansible flow)

**Comment by linuxbandit** [comment]
ok I will put this into the other one

**Comment by linuxbandit** [comment]
this should close itself once it is merged into the other branch of the other PR, and that PR gets merged

**Comment by WikiRik** [comment]
> this should close itself once it is merged into the other branch of the other PR, and that PR gets merged

And with the other one you mean the PR in infrastructure? @linuxbandit

**Comment by linuxbandit** [comment]
no, #166

**Comment by WikiRik** [comment]
So this one can be closed in favor of #166 ?

**Comment by linuxbandit** [comment]
no, it will do automatically. let it be this week

**Comment by linuxbandit** [comment]
In the end I did not do the merge I thought, so it didn't close automatically, closing it now

**Pull request closed without merging**